### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by elkoniu**
Small readme update related to project build issues.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -16,19 +16,31 @@ $ source .venv/bin/activate
...
+To build project you need to be in virtual environment.
+```console
+(.venv) $ make html
+```
 
+Sometimes if you make changes in various places at the same time compiler may keep
+complaining about some non existing error. In this case full project rebuild may
+be needed as documents are usually linked to each others:
 ```console
+(.venv) $ make clean
```
Comment (pbrezina) [inline readme.md:35]: make clean && make html?

**Comment by pbrezina** [comment]
@justin-stephenson Can you please check the language?

File: readme.md (hunk 1 above)
Comment (elkoniu) [reply readme.md]: Updated

File: readme.md [hunk 2]
```diff
@@ -16,19 +16,30 @@ $ source .venv/bin/activate
 (.venv) $ pip3 install -r requirements.txt
 ```
 
-Note: To step out of the virtual environment, run `$ deactivate`.
+If you will finish working in virtual environment you can step out of it:
```
Comment (justin-stephenson) [inline readme.md:19]: s/If you will finish working in virtual environment/When finished working inside the virtual environment

File: readme.md [hunk 3]
```diff
@@ -16,19 +16,30 @@ $ source .venv/bin/activate
...
 
-## Build it
+```console
+(.venv) $ deactivate
+```
 
+## Build it
+To build project you need to be in virtual environment.
```
Comment (justin-stephenson) [inline readme.md:26]: s/build/build the/

and

s/in virtual environment/in the virtual environment/

File: readme.md [hunk 4]
```diff
@@ -16,19 +16,30 @@ $ source .venv/bin/activate
...
 ```console
 (.venv) $ make html
 ```
 
-## View it
+Sometimes if you make changes in various places at the same time compiler may keep
+complaining about some non existing error. In this case full project rebuild may
+be needed as documents are usually linked to each others:
```
Comment (justin-stephenson) [inline readme.md:33]: My suggestion would be `Some changes may require a full project rebuild`

File: readme.md [hunk 5]
```diff
@@ -39,7 +50,10 @@ can view it after the build is finished with:
 $ firefox _build/html/example.html
 ```
 
+Alternative is to use live lookup tools like [online reStructuredText editor].
```
Comment (justin-stephenson) [inline readme.md:53]: s/Alternative/An alternative/

File: readme.md (hunk 2 above)
Comment (elkoniu) [reply readme.md:19]: Fixed

File: readme.md (hunk 3 above)
Comment (elkoniu) [reply readme.md:26]: Fixed

File: readme.md (hunk 4 above)
Comment (elkoniu) [reply readme.md:33]: Fixed

File: readme.md (hunk 5 above)
Comment (elkoniu) [reply readme.md:53]: Fixed

File: readme.md (hunk 4 above)
Comment (justin-stephenson) [reply readme.md]: s/others/other/

**Comment by justin-stephenson** [comment]
One more small change then Ack from me.

File: readme.md (hunk 4 above)
Comment (elkoniu) [reply readme.md]: Fixed

**Comment by justin-stephenson** [comment]
Ack from my side.

**Pull request merged**